docker/m3dbnode/Dockerfile

```diff
@@ -26,5 +26,9 @@ COPY --from=builder /go/src/github.com/m3db/m3/bin/m3dbnode /bin/
 COPY --from=builder /go/src/github.com/m3db/m3/src/dbnode/config/m3dbnode-local-etcd.yml /etc/m3dbnode/m3dbnode.yml
 COPY --from=builder /go/src/github.com/m3db/m3/scripts/m3dbnode_bootstrapped.sh /bin/
 
+# Use setcap and run as specific user
+RUN apk add libcap && \
```

Comment:
Had not seen this before. Did some reading. Kind of bizarre that the capabilities get set on the file/binary level

Reply:
What does the `+ep` do? Can you just add a comment to this line generally also

Reply:
Not opposed to adding a comment, the +e is "effective" and +p is for "permitted".